packages/build-info/src/node/__snapshots__/get-build-info.test.ts.snap

```diff
@@ -53,40 +53,40 @@ exports[`should retrieve the build info for providing a rootDir 1`] = `
   "settings": [
     {
       "baseDirectory": "",
-      "buildCommand": "pnpm --filter @evilcorp/blog... run build",
-      "devCommand": "pnpm --filter @evilcorp/blog run dev",
-      "dist": "packages/blog/dist",
+      "buildCommand": "pnpm --filter @evilcorp/website... run build",
```

Comment:
It looks like we just swapped the order of these two results. Is this test consistently showing up in the new order?